pennylane_lightning_gpu/src/algorithms/AdjointDiffGPU.hpp

```diff
@@ -66,6 +83,42 @@ void applyGeneratorControlledPhaseShift_GPU(SVType &sv,
     sv.applyOperation("P_11", {wires.back()}, adj, {0.0},
                       getP11_CU<decltype(cuUtil::getCudaType(T{}))>());
 }
+template <class T = double, class SVType>
+void applyGeneratorSingleExcitation_GPU(SVType &sv,
+                                        const std::vector<size_t> &wires,
+                                        const bool adj = false) {
+    sv.applyGeneratorSingleExcitation(wires, adj);
+}
+template <class T = double, class SVType>
+void applyGeneratorSingleExcitationMinus_GPU(SVType &sv,
+                                             const std::vector<size_t> &wires,
+                                             const bool adj = false) {
+    sv.applyGeneratorSingleExcitationMinus(wires, adj);
+}
+template <class T = double, class SVType>
+void applyGeneratorSingleExcitationPlus_GPU(SVType &sv,
+                                            const std::vector<size_t> &wires,
+                                            const bool adj = false) {
+    sv.applyGeneratorSingleExcitationPlus(wires, adj);
+}
+template <class T = double, class SVType>
+void applyGeneratorDoubleExcitation_GPU(SVType &sv,
+                                        const std::vector<size_t> &wires,
+                                        const bool adj = false) {
+    sv.applyGeneratorDoubleExcitation(wires, adj);
+}
+template <class T = double, class SVType>
+void applyGeneratorDoubleExcitationMinus_GPU(SVType &sv,
+                                             const std::vector<size_t> &wires,
+                                             const bool adj = false) {
+    sv.applyGeneratorDoubleExcitationMinus(wires, adj);
+}
+template <class T = double, class SVType>
+void applyGeneratorDoubleExcitationPlus_GPU(SVType &sv,
+                                            const std::vector<size_t> &wires,
+                                            const bool adj = false) {
+    sv.applyGeneratorDoubleExcitationPlus(wires, adj);
+}
```

Comment:
I assume the minus and plus operators aren't being deprecated immediately?

Reply:
Ah! I don't think they'll deprecate soon based on PL on-going tasks :slightly_smiling_face: I've added them in PL-Lightning and thought it's good to have them in L-GPU too.